lib/sycamore/sycamore/transforms/extract_graph.py

```diff
@@ -30,6 +34,20 @@ def __init__(self, nodeKey: str, nodeLabel: str, relLabel: str):
         self.relLabel = relLabel
 
 
+class GraphEntity(GraphData):
```

Comment:
Is GraphNode a more generally used convention or entity?

Reply:
Not sure if that would be better since it is technically used to extract a specific type of node, not a singular node.

EntityExtractor uses the label and description of GraphEntity s to extract all nodes that fit that specific label + description.